src/mock_crust/support.rs

```diff
@@ -166,6 +168,13 @@ impl Network {
         self.0.borrow_mut().rng.new_rng()
     }
 
+    /// Return whether handled any message since laster query.
+    pub fn reset_message_handled(&self) -> bool {
```

Comment:
Maybe it should be `sent`, not `handled`; "handling" sounds more like the node receiving the event, rather than the network putting it in the node's queue.